graylog2-web-interface/src/components/users/UserDetails/ProfileSection.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+import * as React from 'react';
+
+import User from 'logic/users/User';
+
+import SectionComponent from '../SectionComponent';
+import ShowConfigValue from '../ShowConfigValue';
+
+type Props = {
+  user: User,
+};
+
+const ProfileSection = ({
+  user: {
+    username,
+    fullName,
+    email,
+    clientAddress,
+    lastActivity,
+    sessionActive,
+  },
+}: Props) => (
+  <SectionComponent title="Profile">
+    <ShowConfigValue label="Username" value={username} />
+    <ShowConfigValue label="Full name" value={fullName} />
+    <ShowConfigValue label="E-Mail Address" value={email} />
+    <ShowConfigValue label="Client Address" value={clientAddress} />
+    <ShowConfigValue label="Last Activity" value={lastActivity} />
+    <ShowConfigValue label="Logged In" value={sessionActive ? 'yes' : 'no'} />
```

Comment:
In a follow up PR we would want a :green_circle:  here :smile:

Reply:
no even better:

:green_circle: Logged in
:red_circle: Not logged in

or smth like that